### PR TITLE
Add support for BE834, 005010X220-BE834, and 005010X220A1-BE834

### PR DESCRIPTION
**Pull request by jakeonfire**
see https://github.com/kputnam/stupidedi/issues/97 (esp. the last comment) for notes.

### CODE REVIEW

**Comment by kputnam** [comment]
This is great, thank you for the hard work! I think your idea to make LS/LE into a table sounds very good. It might take me most of this week to find time to review and merge this.

File: lib/stupidedi/versions/functional_groups/005010/element_defs.rb [hunk 1]
```diff
@@ -921,7 +1148,182 @@ module ElementDefs
...
+          E1218 = t::ID.new(:E1218, "Benefit Status Code"                  , 1, 1,
+            s::CodeList.build(
+              "A" => "Medicare Part A",
+              "B" => "Medicare Part B",
+              "C" => "Medicare Part A and B",
+              "D" => "Medicare",
+              "E" => "No Medicare"))
+
+          E1219 = t::ID.new(:E1219, "Consolidated Omnibus Budget Reconciliation
+Act (COBRA) Qualifying", 1, 2,
```
Comment (kputnam) [inline lib/stupidedi/versions/functional_groups/005010/element_defs.rb]: This seems to be an unintentional line break?

**Comment by jakeonfire** [comment]
fixed the rogue newline

**Comment by kputnam** [comment]
Thanks again, and this will be included in the next release.

**Pull request merged**